calico_containers/pycalico/util.py

```diff
@@ -1,16 +1,20 @@
-from netaddr.core import AddrFormatError
+#!/usr/bin/python
+"""General utility functions"""
 
-import netaddr
 import socket
 import sys
 import os
 import re
 import logging
-from netaddr import IPNetwork, IPAddress
 from subprocess import check_output, CalledProcessError
 
-_log = logging.getLogger(__name__)
-_log.addHandler(logging.NullHandler())
+import netaddr
+from netaddr import IPNetwork, IPAddress
+from netaddr.core import AddrFormatError
+
+
+LOG = logging.getLogger(__name__)
```

Comment:
I know LOG is quite widely used here, but the rest of the code uses _log, and I'd prefer to keep it internally consistent - can we please keep it as it was?